packages/example_hardhat/hardhat.config.ts

```diff
@@ -25,43 +15,5 @@ const config: HardhatUserConfig = {
       },
     ],
   },
-  namedAccounts: {
-    // *namedAccounts should match packages/core/hardhat.config.ts if using existing roles*
-    // new namedAccounts may be added at the end of this list as required
-    //------EXAMPLE NAMED ACCOUNTS FROM CORE---------------------------------------//
-    deployer: {
-      default: 1,
-    }, // deploy contracts and make sure they are set up correctly
-    sandAdmin: {
-      default: 2,
-    }, // can add super operators and change admin
-    backendAuthWallet: {
-      // default is computed from private key:
-      // "0x4242424242424242424242424242424242424242424242424242424242424242"
-      default: '0x17c5185167401eD00cF5F5b2fc97D9BBfDb7D025',
-    },
-    //------NEW NAMED ACCOUNTS HERE------------------------------------------------//
-  },
-  defaultNetwork: "hardhat",
-  networks: {
-    hardhat: {
-      forking: {
-        enabled: false, // * note: if set to true then CI will fail *
-        blockNumber: 16000000,
-        url: "http://localhost:8545",
-      },
-      loggingEnabled: false,
-      chainId: 1337,
-      allowUnlimitedContractSize: false,
-      mining: {
-        auto: true,
-        interval: 1000,
-        mempool: {
-          order: "fifo",
-        },
-      },
-    },
-  },
```

Comment:
I want to reinstate this section as I am seeing some problems with this set up in recent PRs...

Reply:
We don't need named accounts this is a hardhat-deploy feature and hardhat manual doesn't mention it.

Reply:
We need something that explains how to replicated namedAccounts for testing.
For example, if we need to set up a sandAdmin or a backendAuthWallet for tests. Can you add something in your example tests maybe that shows this?

Reply:
https://hardhat.org/tutorial/testing-contracts#writing-tests